packages/reporter/src/main.tsx

```diff
@@ -68,7 +68,7 @@ class Reporter extends Component<SingleReporterProps> {
       <div className={cs(className, 'reporter', {
         'studio-active': appState.studioActive,
       })}>
-        {renderReporterHeader({ appState, statsStore })}
+        {renderReporterHeader({ appState, statsStore, spec: this.props.runnerStore.spec! })}
```

Comment:
Is `!` a good idea here? Should we be more defensive? What is `spec` is undefined, does something bad happen?

Reply:
I don't think we ever render the reporter unless we have a spec chosen so this should always be defined. Will clean this up though, depending on how/if I change up the testFilter communication.